rama-http/src/layer/validate_request/validate_request_header.rs

```diff
@@ -90,7 +90,7 @@ where
 /// See the [module docs](crate::layer::validate_request) for an example.
 pub struct ValidateRequestHeader<S, T> {
     inner: S,
-    validate: T,
+    pub(crate) validate: T,
```

Comment:
Changed this so that I could access it in `require_authorization` with the intention of avoiding circular dependency between this module and `require_authorization` (if I had implemented the `x_allow_anonymous` functions here)

Could expose this through a method if preferable.

Reply:
No this is fine. It's still not part of public rama-http api so this is ok as-is. We still have freedom to change it easily this way. Well done.